docs/rest_api.rst

```diff
@@ -663,3 +663,91 @@ be rejected.
   :statuscode 404: If the channel does not exist
   :statuscode 409: If the given block number argument is invalid
   :statuscode 500: Internal Raiden node error
+
+
+.. http:get:: /api/1/channels/history/(token_address)/(partner_address)
```

Comment:
Can we express that `(partner_address)` is optional? If not it may be better to have a section for each URL

Reply:
I forgot, thanks.